x-pack/plugins/security_solution/public/management/pages/policy/view/ingest_manager_integration/endpoint_policy_create_multi_step_extension.tsx

```diff
@@ -6,13 +6,110 @@
  */
 
 import React, { memo } from 'react';
-import type { PackagePolicyCreateMultiStepExtensionComponentProps } from '@kbn/fleet-plugin/public';
+import { EuiText, EuiIcon, EuiFlexGroup, EuiFlexItem, EuiLink, EuiSpacer } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import styled from 'styled-components';
 
 /**
- * TBD: A component to be displayed in multi step onboarding process.
+ * A component to be displayed in multi step onboarding process.
  */
-export const EndpointPolicyCreateMultiStepExtension =
-  memo<PackagePolicyCreateMultiStepExtensionComponentProps>(({ newPolicy }) => {
-    return <></>;
-  });
+export const EndpointPolicyCreateMultiStepExtension = memo(() => {
+  return (
+    <Container>
+      <LargeCustomSpacer />
+      <Title />
+      <MediumCustomSpacer />
+
+      <EuiFlexGroup>
+        <EuiFlexItem grow={false} style={{ alignItems: 'center' }}>
+          <LargeSecurityLogo />
+        </EuiFlexItem>
+
+        <EuiFlexItem>
+          <div>
+            <Features />
+          </div>
+        </EuiFlexItem>
+      </EuiFlexGroup>
+
+      <MediumCustomSpacer />
+      <Details />
+      <LargeCustomSpacer />
+    </Container>
+  );
+});
 EndpointPolicyCreateMultiStepExtension.displayName = 'EndpointPolicyCreateMultiStepExtension';
+
+const Container = styled('div')`
+  padding: 0 23px;
+`;
+
+const Title = () => (
+  <EuiText>
+    <h3>
+      <FormattedMessage
+        id="xpack.securitySolution.endpoint.policy.multiStepOnboarding.title"
+        defaultMessage="We'll save your integration with our recommended defaults."
+      />
+    </h3>
+  </EuiText>
+);
+
+const LargeCustomSpacer = () => (
+  <>
+    <EuiSpacer size="m" />
+    <EuiSpacer size="xxl" />
+  </>
+);
+
+const MediumCustomSpacer = () => (
+  <>
+    <EuiSpacer size="m" />
+    <EuiSpacer size="xl" />
+  </>
+);
```

Comment:
Please merge these two separate components into a single one called `CustomSpacer` that takes sizes as an array and then spits out  `EuiSpacers` based on that. Then you can just use something like  `<CustomSpacer spacers={['m', 'xxl']} />` or `<CustomSpacer spacers={['m', 'xl']} />` where needed.